app/src/main/java/io/heckel/ntfy/msg/NotificationService.kt

```diff
@@ -415,11 +443,14 @@ class NotificationService(val context: Context) {
 
         private const val TAG = "NtfyNotifService"
 
-        private const val CHANNEL_ID_MIN = "ntfy-min"
-        private const val CHANNEL_ID_LOW = "ntfy-low"
-        private const val CHANNEL_ID_DEFAULT = "ntfy"
-        private const val CHANNEL_ID_HIGH = "ntfy-high"
-        private const val CHANNEL_ID_MAX = "ntfy-max"
+        private const val DEFAULT_CHANNEL = "ntfy"
+        private const val DEFAULT_GROUP_ID = "ntfy"
+
+        private const val PRIORITY_MIN = "-min"
+        private const val PRIORITY_LOW = "-low"
+        private const val PRIORITY_DEFAULT = ""
+        private const val PRIORITY_HIGH = "-high"
+        private const val PRIORITY_MAX = "-max"
```

Comment:
Feels a little bit hacky to use those constants as suffixes for the channel names, wdyt?